Sources/Commands/APIDigester.swift

```diff
@@ -64,18 +64,22 @@ struct APIDigesterBaselineDumper {
     }
 
     /// Emit the API baseline files and return the path to their directory.
-    func emitAPIBaseline(for modulesToDiff: Set<String>) throws -> AbsolutePath {
+    func emitAPIBaseline(for modulesToDiff: Set<String>,
+                         baselineDir: AbsolutePath?,
+                         force: Bool) throws -> AbsolutePath {
         var modulesToDiff = modulesToDiff
         let apiDiffDir = inputBuildParameters.apiDiff
-        let baselineDir = apiDiffDir.appending(component: baselineTreeish)
+        let baselineDir = (baselineDir ?? apiDiffDir).appending(component: baselineTreeish)
         let baselinePath: (String)->AbsolutePath = { module in
             baselineDir.appending(component: module + ".json")
         }
 
-        for module in modulesToDiff {
-            if localFileSystem.exists(baselinePath(module)) {
-                // If this baseline already exists, we don't need to regenerate it.
-                modulesToDiff.remove(module)
+        if !force {
+            for module in modulesToDiff {
+                if localFileSystem.exists(baselinePath(module)) {
+                    // If this baseline already exists, we don't need to regenerate it.
+                    modulesToDiff.remove(module)
```

Comment:
You can also use .filter instead (of this for loop); remove can incur a perf cost.